.github/CODEOWNERS

```diff
@@ -4,4 +4,4 @@
 # the repo. Unless a later match takes precedence,
 # these owners will be requested for review when someone
 # opens a pull request.
-*       @dav3r @felddy @jsf9k @mcdonnnj @cisagov/team-ois
+* @dav3r @felddy @hillaryj @jsf9k @mcdonnnj
```

Comment:
This list can probably be trimmed to @dav3r and @jsf9k and maybe now @mcdonnnj. 😄

Reply:
Good point, trimmed in 6bc9489.